code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -80,3 +80,6 @@
 
 	var/list/splinted_limbs = list() //limbs we know are splinted
 	var/original_eye_color = "#000000"
+
+	/// How many teeth does our human have?
+	var/teeth = 32
```

Comment:
This might make more sense to be on the head bodypart, that way you can also easily modify the number per species